R/extract_ngram.R

```diff
@@ -9,9 +9,37 @@
 #'
 #' @examples
 #' tweets <- c(
-#'    "Make America Great Again DonaldTrump"
+#'    "Make", "America", "Great", "Again", "DonaldTrump"
 #' )
 #' extract_ngram(tweets, n=3)
 extract_ngram <- function(tweets, n = 3) {
+  if (length(tweets) < n) {
+    stop("length of ngrams should be less than number of words in vector of tweets")
+  }
+  if(!is.character(tweets)) {
+    stop("input should be a character vector")
+  }
 
+  if (length(tweets) < n) {
+    stop("length of ngrams should be less than length of tweets")
+  }
+  if(!is.character(tweets)) {
+    stop("input should be a character vector")
+  }
+
+  # tweets_str <- paste(tweets, sep=" ", collapse = NULL)
```

Comment:
Could you please delete this line?

Reply:
Sure. Missed removing the commented lines earlier. Deleted now.